model/mustache.go

```diff
@@ -0,0 +1,79 @@
+package model
+
+import (
+	"fmt"
+	"sort"
+
+	"github.com/hpcloud/fissile/mustache"
+)
+
+// GetVariablesForRole returns all the environment variables required for
+// calculating all the templates for the role
+func (r *Role) GetVariablesForRole() ([]*ConfigurationVariable, error) {
```

Comment:
I don't think we use that pattern anywhere in fissile. Any reason to use it here?